docs/basisterminologie.md

```diff
@@ -0,0 +1,184 @@
+
+In sommige gevallen bestaat de inhoud van een klantcontact uit verschillende onderdelen, die bovendien verschillende vastleggingsvormen kunnen vereisen. Denk bijvoorbeeld aan een ontvangen e-mailbericht (‘platte’ tekst) met bijlagen (documenten). In andere gevallen kan het behulpzaam zijn om dezelfde inhoud in meerdere vormen (‘representaties’) vast te leggen. Zo kan het, om verdere geautomatiseerde verwerking mogelijk te maken, enerzijds raadzaam zijn de in een e-formulier ingevulde gegevens als gestructureerde data te registreren, terwijl het vanuit dienstverleningsoogpunt wenselijk is diezelfde gegevens in de vorm van een Pdf-bestand aan de indiener ter beschikking te stellen.
+
+Om te waarborgen dat de standaard aan dit soort scenario's invulling kan geven, wordt de inhoud van een klantcontact vastgelegd in één of meerdere ‘inhoudsdelen’, die ieder één van de volgende drie vormen kunnen hebben:
+
+1. (‘Platte’) tekst die (een deel van) de inhoud van het klantcontact beschrijft.
+2. Een verwijzing naar het informatieobject (of document) dat (een deel van) de inhoud van het klantcontact beschrijft.
+3. Een verwijzing naar het dataobject dat (een deel van) de inhoud van het klantcontact beschrijft. De standaard legt geen beperkingen op als het gaat om de objecttypen (en bijbehorende registers) waarnaar vanuit een klantcontact verwezen mag worden, maar erkent expliciet het bestaan van het [register voor overige objectregistraties](https://vng.nl/projecten/overige-objecten-registratie-api).
+
+<img src="./assets/basisterminologie/klantinteracties-submodel-inhoud-klantcontact.svg" alt="Submodel inhoud van het klantcontact" width="40%"/>
+
+### Onderwerp van het klantcontact
+
+Naast inhoud heeft het klantcontact een onderwerp. Omdat (zeker tijdens een gesprek) uiteenlopende zaken besproken kunnen worden, kunnen dit ook meerdere onderwerpen zijn. Het onderwerp of de onderwerpen beschrijft waar het klantcontact over ging. Als het klantcontact een eenvoudig verzoek om informatie betrof (“wat zijn de voorwaarden voor kwijtschelding van gemeentelijke belastingen?”), zal zo’n onderwerp vaak een generiek karakter hebben.
+
+Maar het onderwerp kan ook meer specifiek of persoonlijk zijn en betrekking hebben op een specifiek voor één klant gestart proces (“kunt u toelichting geven bij het verzoek extra informatie aan te leveren om de aanvraag van mijn omgevingsvergunning te kunnen beoordelen?”). Om in verschillende situaties onderwerp(en) van het klantcontact te kunnen vastleggen, worden onderwerpdelen geregistreerd. Net als de klantcontactinhoud, kunnen deze kunnen verschillende vormen hebben:
+
+1. (‘Platte’) tekst die (een deel van) het onderwerp van het klantcontact beschrijft.
+2. Een verwijzing naar de interne taak die aanleiding gaf tot (een deel van) het klantcontact.
+3. Een verwijzing naar de zaak waarover (een deel van) het klantcontactging.
+4. Een verwijzing naar de klanttaak waarover (een deel van) het klantcontact ging.
```

Comment:
Een klantcontact kan ook over een vorig klantcontact gaan. Moet er niet een verwijzing toegevoegd worden van onderwerpdeel naar een klantcontact ? Dit is weliswaar commentaar op het informatiemodel, maar het kwam hier naar boven....

Reply:
Deze relatie hadden we volgens mij specifiek niet erkend omdat we ervan uitgaan dat opvolgende klantcontacten verbonden zijn middels een taak of zaak. Bij nader inzien gaat dit niet altijd op: een inwoner kan terugbellen over (de inhoud van) een eerder klantcontact, terwijl de daarbij betrokken medewerker geen reden heeft gezien voor opvolging en dus geen taak heeft aangemaakt. Vraag is of we deze relatie in het informatiemodel ook willen opnemen met het doel die registreerbaar te maken?